pkgs/development/python-modules/jeepney/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pytest
+, testpath
+, tornado_4
+}:
+
+buildPythonPackage rec {
+  pname = "jeepney";
+  version = "0.4";
+
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0w1w1rawl9k4lx91w16d19kbmf1349mhy8ph8x3w0qp1blm432b0";
+  };
+
+  checkInputs = [
+    pytest
+    testpath
+    tornado_4
```

Comment:
Where is this requirement specified?
It's really sad to see people adding different versions of python packages to `python-packages.nix` again and again. (I know it wasn't you).

Reply:
> Where is this requirement specified?

My mistake, this should be `tornado` and a `propagatedBuildInputs`.
Seems to be a sort of integration https://gitlab.com/takluyver/jeepney/blob/master/docs/integrate.rst

> It's really sad to see people adding different versions of python packages to python-packages.nix again and again. (I know it wasn't you).

With enough frequency we should probably add a check to OfBorg for it.